coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -33,3 +41,107 @@ def __init__(self, documentation, language, docstyle,
 
     def __str__(self):
         return self.documentation
+
+    def parse_documentation(self):
+        """
+        Parses documentation independent of language and docstyle.
+
+        :return:
+            A list of the parsed documentation metadata.
+        :raises:
+            NotImplementedError
+        """
+        if self.language == "python":
+            return self.parse_python()
+        else:
+            raise NotImplementedError(
+                "Documentation parsing for {} in {}"
+                " has not been implemented yet".format(
+                    self.language, self.docstyle))
+
+    def parse_python(self):
+        """
+        Parse documentation. Usable attributes are:
+
+        ``:param``
+        ``@param``
+
+        ``:return``
+        ``@return``
```

Comment:
yes I initially thought of supporting default and doxygen altogether, but that wont work. So yeah, I will have to change some things. :+1: